awesomplete.js

```diff
@@ -265,6 +274,10 @@ _.prototype = {
 		else {
 			this.close();
 		}
+	},
+
+	setStatus: function(message) {
+		this.status.innerHTML = message;
```

Comment:
We need to use either `textContent` for botth, or `innerHTML` for both. Getting `textContent` and then setting `innerHTML` could result in angle brackets being parsed as HTML etc.
Also, I disagree with using a separate function just for this one line.